src/native-common/View.tsx

```diff
@@ -69,29 +69,61 @@ export class View extends ViewBase<Types.ViewProps, {}> {
                 this._internalProps.pointerEvents = 'box-none';
             }
         }
+
+        if (RN.Platform.OS === 'windows') {
+            this._processDragAndDropProps();
+        }
+    }
+
+    private _processDragAndDropProps() {
+        for (const name of ['onDragEnter', 'onDragOver', 'onDrop', 'onDragLeave']) {
+            const handler = this._internalProps[name];
+
+            if (handler) {
+                this._internalProps.allowDrop = true; // TODO: let RN figure this out?
```

Comment:
If this is just a reminder that we may want to revisit it in the future, don't make it a "TODO". If it's something that needs to be fixed, leave it as a TODO but add a reference to a bug report.